osutil/mountinfo.go

```diff
@@ -46,6 +48,49 @@ type MountInfoEntry struct {
 	SuperOptions   map[string]string
 }
 
+func flattenMap(m map[string]string) string {
+	keys := make([]string, 0, len(m))
+	for key := range m {
+		keys = append(keys, key)
+	}
+	sort.Strings(keys)
+	var buf bytes.Buffer
+	for i, key := range keys {
+		if i > 0 {
+			fmt.Fprintf(&buf, ",")
+		}
+		if m[key] != "" {
+			fmt.Fprintf(&buf, "%s=%s", key, m[key])
+		} else {
+			fmt.Fprintf(&buf, "%s", key)
+		}
+	}
+	return buf.String()
+}
+
+func flattenList(l []string) string {
```

Comment:
Why not use `strings.Join(l, ",")`?

Reply:
Hmm, because I'm silly :-) EDIT: Will do